packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -124,7 +124,10 @@ const EntityLayoutWrapper = (props: { children?: ReactNode }) => {
 
   return (
     <>
-      <EntityLayout UNSTABLE_extraContextMenuItems={extraMenuItems}>
+      <EntityLayout
+        UNSTABLE_extraContextMenuItems={extraMenuItems}
+        UNSTABLE_contextMenuOptions={{ disableUnregister: true }}
```

Comment:
Maybe let's not change the behaviour fo the `example-app` for contributors.

Reply:
updated